### PR TITLE
Use react-native link instead of rnpm link

### PR DESCRIPTION
**Pull request by 7kfpun**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -17,11 +17,14 @@ A react-native wrapper for handling in-app purchases.
 
 ### Add it to your project
 
-1. Make sure you have `rnpm` installed: `npm install rnpm -g`
+1. Install and link the react-native-in-app-utils package:
 
-2. Install with rnpm: `rnpm install react-native-in-app-utils`
+```
+react-native install react-native-in-app-utils
```
Comment (chirag04) [inline Readme.md]: npm install?

Comment (7kfpun) [inline Readme.md]: It comes with react-native-cli.

Comment (chirag04) [inline Readme.md]: i would prefer npm install please

Comment (kanerogers) [inline Readme.md]: What's the thinking here?

Comment (mfikes) [inline Readme.md]: If `npm install` is used, `react-native link` would also need to be used. (See https://stackoverflow.com/a/43728793/4284484)

Comment (chirag04) [inline Readme.md]: my thinking here was that rnpm and rn-link both had some pain points. changes of getting it right with manual installation is way more than doing it with rn-link.

Happy to roll with rn-link if you guys think it works fine. also needs rebase.

**Comment by philwilks** [comment]
This change has already been made in another commit

**Pull request closed without merging**